packages/babel-helpers/src/helpers.js

```diff
@@ -585,3 +599,18 @@ helpers.toConsumableArray = template(`
     }
   });
 `);
+
+helpers.toPropertyKey = template(`
+  (function (key) {
+    if (typeof key === "symbol" || (
+      typeof key === "object" && 
+      key !== null &&
+      typeof key.toString === "function" && 
+      key.toString().slice(0, 6) === "Symbol(")) 
```

Comment:
I wanted to add a test for the case of polyfilled symbol, but unfortunately can't find a straightforward way to do it :( anyways, this is fixed in latest push